app/code/local/Magmodules/Webwinkelconnect/Helper/Data.php

```diff
@@ -69,4 +69,29 @@ public function getHtmlStars($rating)
         return $html;
     }
 
+    public function getApiKey(): string {
+        return Mage::getStoreConfig('webwinkelconnect/general/api_key');
+    }
+
+    public function getShopId(): string {
+        return Mage::getStoreConfig('webwinkelconnect/general/api_id');
+    }
+
+    public function isProductReviewInviteEnabled(): bool {
+        if (!Mage::getStoreConfig('webwinkelconnect/general/enabled')) {
+            return false;
+        }
+        if (!Mage::getStoreConfig('webwinkelconnect/product_review_invites/enabled')) {
+            return false;
+        }
+        if ($this->getApiKey() == '') {
```

Comment:
With the above changes, the checks for API key and shop ID will become `=== null`.